p2p/conn/connection.go

```diff
@@ -553,6 +572,21 @@ func (c *MConnection) sendPacketMsg() bool {
 	return false
 }
 
+func (c *MConnection) processFullMsg() {
+FOR_LOOP:
+	for {
+		select {
+		case <-c.quitProcessFullMsg:
+			break FOR_LOOP
+		case msg := <-c.receivedFullMsg:
+			c.onReceive(msg.chID, append([]byte(nil), msg.msgBytes...))
+		default:
+			// never block
```

Comment:
If you never block this routine will become a busy loop wasting cpu. Whats the point?

Reply:
Thank you for your interest in the PR! You're correct; the code currently includes busy waiting, which I plan to remove in the final version. This was introduced as part of an experimental debugging process on this branch to diagnose the cause of the failing e2e tests. Nevertheless, the busy waiting is not the reason for the test failures. It was added solely to assess whether it would alter the outcome of the tests.